hikari/traits.py

```diff
@@ -124,23 +124,22 @@ def proxy_settings(self) -> config.ProxySettings:
 
 
 @typing.runtime_checkable
-class DispatcherAware(typing.Protocol):
-    """Structural supertype for a dispatcher-aware object.
+class EventAware(typing.Protocol):
```

Comment:
`EventManagerAware`?

Reply:
seems a bit redundant but if you prefer that name then sure

Reply:
On the other hand it may also make sense to add methods like "subscribe" here like we do on ShardAware but that wouldn't be necessary if we're renaming it to EventManagerAware